src/services/user.service.js

```diff
@@ -0,0 +1,39 @@
+const { User } = require('../models/user.model');
+
+const create = async (user) => {
+  return User.create(user);
```

Comment:
Consider adding validation for the `user` object to ensure it meets the expected format and constraints before attempting to create a new user.